e2e/server_test.go

```diff
@@ -382,3 +383,60 @@ func testProxyMemConfigClientCanGetAndPatch(t *testing.T, v2Enabled bool) {
 	require.NoError(t, err)
 	require.Equal(t, cfg.PutLatency, expectedChange)
 }
+
+// TestInterleavedVersions alternately disperses payloads to v1 and v2, and then retrieves them.
+func TestInterleavedVersions(t *testing.T) {
+	t.Parallel()
```

Comment:
this would be a good candidate test for fuzzing, or at least large scale random generation. Like instead of just testing 4 dispersals, generate a ton, or even better let fuzzer generate them (having `useV2` as a fuzzer input variable).

Reply:
yep brainfart of my part, you're right.